common/chains/client/types/node.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/v2/common/types"
+)
+
+type NodeClientAPI[
```

Comment:
Does this represent the subset of client methods required to be a 'node' in the pool? What about something simpler, like `PoolNode`?

Reply:
I think we are trying to move away from the pool naming since now the component that handles all that is the `MultiNodeClient`. How about `NodeClient`, and `SendOnlyClient` accordingly?